src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -31,6 +31,7 @@ namespace AppInstaller::Runtime
         constexpr std::string_view s_PortablePackageRoot = "WinGet"sv;
         constexpr std::string_view s_PortablePackagesDirectory = "Packages"sv;
         constexpr std::string_view s_LinksDirectory = "Links"sv;
+        constexpr std::string_view s_CheckpointsDirectory = "Checkpoints"sv;

```

Comment:
At some point I think we will need to have something going through and cleaning out old resume context data.  Doesn't need to be now.